misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/JBallerinaDebugServer.java

```diff
@@ -179,7 +179,7 @@ ClientConfigHolder getClientConfigHolder() {
         // Todo - Implement
         capabilities.setSupportsCompletionsRequest(false);
         capabilities.setSupportsRestartRequest(false);
-        capabilities.setSupportsConditionalBreakpoints(false);
+        capabilities.setSupportsConditionalBreakpoints(true);
```

Comment:
Lets move up this line to the `supported capabilities` list.

Reply:
Done.